source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ADCIOSAdaptiveHostConfig.h

```diff
@@ -0,0 +1,31 @@
+//
+//  ADCIOSAdaptiveHostConfig.h
+//  ADCIOSFramework
+//
+//  Created by jwoo on 6/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+#import "TextBlock.h"
+#import "Image.h"
+#import "HostConfig.h"
+
+using namespace AdaptiveCards;
+
+@interface ADCIOSAdaptiveHostConfig:NSObject
```

Comment:
Good point, apple requires at least 4 letters prefix to prevent name collisions with each others.
I couldn't come up with additional one letter.